spec/features/user_profile_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+# As an authenticated user
+# I want to see what I have contributed
+
+# Acceptance Criteria:
+# [ ] There is a profile option
+# [ ] I can click an edit credentials link
+
+feature "sees profile in top bar" do
+  let!(:user) { User.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.